src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -318,7 +319,7 @@ public void waitForValueToBePresentInElementsAttributeByCss(String selector, Str
     changeImplicitWait(250, TimeUnit.MILLISECONDS);
     try {
       waitFor.until(CommonExpectedConditions.valueToBePresentInElementsAttribute(
-          By.cssSelector(selector), attribute, value));
+          By.cssSelector(selector), attribute, escapeHtml(value)));

```

Comment:
I don't think we should escape on this level, we should explicitly escape title of a video when passing to a method, not in here. Let's move it to `verifyVideoAdded(String videoTitle)` in `SpecialVideosPageObject`